### PR TITLE
feat: add treasury drawer item

### PR DESCRIPTION
**Pull request by rickstaa**
This commit introduces the Treasury drawer item for mobile devices, enhancing user navigation to the treasury page. It also fixes the active proposals indicator shown on the desktop version.

I currently implemented the treasury page as a standalone item for minimal code changes; an alternative could be to group it with governance features in a submenu. Feedback on the preferred approach is welcome.

**Visual Preview**

image

### CODE REVIEW

**Comment by rickstaa** [comment]
If we decide to merge the two under a submenu, we can rename the `voting` page to `governance`.

https://github.com/livepeer/explorer/blob/a16b5343c7209bbe584261dd6b0b242b1c676990/layouts/main.tsx#L185-L186

File: layouts/main.tsx [hunk 1]
```diff
@@ -483,15 +505,15 @@ const Layout = ({ children, title = "Livepeer Explorer" }) => {
                               }}
                             >
                               Treasury{" "}
-                              {(totalActiveTreasuryProposals ?? 0) > 0 && (
+                              {(totalActiveProposals ?? 0) > 0 && (
```
Comment (victorges) [inline layouts/main.tsx:508]: Is this a typo/unresolved conflict? I thought the variable was named `totalActiveTreasuryProposals` now

**Review by victorges — APPROVED** [review]
LGTM there's only a potential typo

File: layouts/main.tsx (hunk 1 above)
Comment (rickstaa) [reply layouts/main.tsx]: @victorges, good catch. I think that one was introduced by git when I rebased my previous branch onto the main branch 🤔. I just fixed it 👍🏻.

**Comment by rickstaa** [comment]
> LGTM there's only a potential typo

@victorges, you don't want to rename the `voting` page for now but only change the mobile tab text? Are you also okay with having the same icon with the `treasury` and `governance` items? We can also change the `treasury` icon to something else 🤔.

**Review by victorges — APPROVED** [review]
LGTM

**Pull request merged**